<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: black;
}

QLabel {
	color: white;
	font-size: 42px;
	font-family: arial;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#BEBEBE;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #BEBEBE, stop:1 #F5F5F5);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plusminus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#BEBEBE;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #BEBEBE, stop:1 #F5F5F5);
}</string>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#BEBEBE;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #BEBEBE, stop:1 #F5F5F5);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_divide">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#FF9900;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FF9900, stop:1 #FFC56D);
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_subtract">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#FF9900;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FF9900, stop:1 #FFC56D);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_multiply">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#FF9900;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FF9900, stop:1 #FFC56D);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#FF9900;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FF9900, stop:1 #FFC56D);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#808080;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #808080, stop:1 #BEBEBE);
}</string>
    </property>
    <property name="text">
     <string>•</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_calculate">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  	#FF9900;
	border-style: solid;
	border-color: black;
	border-width: 1px;
	border-radius: 30px;
	font-size: 24px;
	font-family: arial;
}

QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,stop:0 #FF9900, stop:1 #FFC56D);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
